bindings/python/example_scripts/gncinvoice_jinja.py

```diff
@@ -150,8 +151,8 @@ def main(argv=None):
 
     # Try to open the given input
     try:
-        print("Opening", input_url, ".")
-        session = gnucash.Session(input_url, ignore_lock=ignore_lock)
+        print("Opening", input_url, " (ignore-lock = read-only)." if ignore_lock else ".")
+        session = gnucash.Session(input_url, SessionOpenMode.SESSION_READ_ONLY if ignore_lock else SessionOpenMode.SESSION_NORMAL_OPEN)
```

Comment:
Can you do something about the line length here, maybe make a `mode` variable on a separate line?